CoreScriptsRoot/Modules/DeveloperConsoleModule.lua

```diff
@@ -2719,12 +2719,16 @@ end
 -- Permissions --
 -----------------
 do
-	local permissions;
+	local permissionsLoading, permissions = false;
 	function DeveloperConsole.GetPermissions()
+		while permissionsLoading do wait() end
```

Comment:
hmm alright, I guess permissions could change in the middle of a session, but seems unnecessary to do this more than once, as it seems to complicate the code quite a bit. I'm ok with this change for now, but we should consider moving the permission check to script start up